File: Java/ex5.java
// Escreva um algoritmo que receba a idade, a altura e o peso de 25 pessoas. Calcule e exiba.
// a) Quantidade de pessoas com idade superior a 50 anos.
// b) A média das alturas das pessoas com idade entre 10 e 20 anos;
// c) A porcentagem de pessoas com peso inferior a 40 quilos entre todas as analisadas.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int count50plus = 0;
        double somaAlturas = 0;
        int countAlturas = 0;
        int count40minus = 0;

        for (int i = 0; i < 25; i++) {
            System.out.println("Digite a idade:");
            int idade = scanner.nextInt();
            System.out.println("Digite a altura:");
            double altura = scanner.nextDouble();
            System.out.println("Digite o peso:");
            double peso = scanner.nextDouble();

            if (idade > 50) {
                count50plus++;
            }
            if (idade >= 10 && idade <= 20) {
                somaAlturas += altura;
                countAlturas++;
            }
            if (peso < 40) {
                count40minus++;
            }
        }

        System.out.println("Quantidade de pessoas com mais de 50 anos: " + count50plus);
        System.out.println("Média das alturas das pessoas entre 10 e 20 anos: " + (countAlturas > 0 ? somaAlturas / countAlturas : 0));
        System.out.println("Porcentagem de pessoas com peso inferior a 40kg: " + (count40minus / 25.0 * 100) + "%");
    }
}

File: Python/ex5.py
# Escreva um algoritmo que receba a idade, a altura e o peso de 25 pessoas. Calcule e exiba.
# a) Quantidade de pessoas com idade superior a 50 anos.
# b) A média das alturas das pessoas com idade entre 10 e 20 anos;
# c) A porcentagem de pessoas com peso inferior a 40 quilos entre todas as analisadas.

count_acima_50 = 0
total_altura_10_20 = 0
count_10_20 = 0
count_inferior_40kg = 0
for i in range(25):
    idade = int(input(f"Digite a idade da {i+1}ª pessoa: "))
    altura = float(input(f"Digite a altura da {i+1}ª pessoa (em metros): "))
    peso = float(input(f"Digite o peso da {i+1}ª pessoa (em kg): "))
    if idade > 50:
        count_acima_50 += 1
    if 10 <= idade <= 20:
        total_altura_10_20 += altura
        count_10_20 += 1
    if peso < 40:
        count_inferior_40kg += 1
media_altura = total_altura_10_20 / count_10_20 if count_10_20 > 0 else 0
porcentagem_under_40kg = (count_inferior_40kg / 25) * 100
print("Quantidade de pessoas com idade superior a 50 anos:", count_acima_50)
print("Média das alturas das pessoas com idade entre 10 e 20 anos:", media_altura)
print("Porcentagem de pessoas com peso inferior a 40 quilos:", porcentagem_under_40kg)
